packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1011,6 +1011,36 @@ export const components = ({
         }
     `,
     },
+    MuiDrawer: {
+      styleOverrides: {
+        root: {
+          height: "100%",
+        },
+        paper: ({ ownerState }) => ({
+          width: ownerState.variant === "temporary" ? "400px" : "360px",
```

Comment:
Should these widths be defined elsewhere? I wanted to err on the side of not over-optimizing but open to suggestions!

Reply:
I agree, we should _probably_ have this in some sort of constant somewhere

Reply:
We currently don't support multiple sizes as an option but each variant has it's own width (400 or 360). 

I was thinking through options for this earlier and I tried to pass in the sizes as an exported variable from `Drawer.tsx` to `Components.tsx` but am not sure if that's possible (seems like I'd need to import it from the `@okta/odyssey-react-mui/labs`) package directly but I don't believe that is possible in `components.tsx`. Is there another way that might be better option you know of @chrispulsinelli-okta @bryancunningham-okta?